dashboard/package.json

```diff
@@ -17,7 +17,7 @@
     "classnames": "^2.3.1",
     "crypto-js": "^4.1.1",
     "customize-cra": "^1.0.0",
-    "dotenv": "^16.0.1",
+    "dotenv": "^16.0.1",    
```

Comment:
Can we get rid of the trailing whitespace here?

Reply:
Removed the trailing whitespace